processingchain/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/ProcessingChainBDTopoTest.groovy

```diff
@@ -58,10 +52,39 @@ class ProcessingChainBDTopoTest extends ChainProcessAbstractTest{
         }
     }
 
+    def loadFiles(String inseeCode){
+        H2GIS h2GISDatabase = H2GIS.open(h2db, "sa", "")
```

Comment:
Be careful, by using a fixed String as database name, you will always use the same database for each tests. If needed, to avoid conflict, you might have to use random names (by using `UUID` class).

Reply:
OK I have corrected that using specific name for each test (to be able to easily find the right db when applying a test...)